features/ProfileTab/PreferencesScreen/CurrencyDropdown/CurrencyDropdown.tsx

```diff
@@ -0,0 +1,39 @@
+import { useContext, useState } from "react";
+import { Dropdown } from "react-native-element-dropdown";
+import { Surface, Text } from "react-native-paper";
+
+import SettingsContext from "../../../../contexts/SettingsContext";
+import Currency from "../../../../types/Currency";
+
+const data: {
+  label: string;
+  value: Currency;
+}[] = [
+  { label: "Dollar", value: "USD" },
```

Comment:
label: "US Dollar" as above

Reply:
Fixed in [fd48dd3](https://github.com/flatly-pw/mobile-app/pull/13/commits/fd48dd317fdca05be37b7145fffa014f31b955d2)